pkg/stub/handler.go

```diff
@@ -114,6 +115,12 @@ type Handler struct {
 }
 
 func (h *Handler) StatusUpdate(condition v1alpha1.SamplesResourceConditionType, srcfg *v1alpha1.SamplesResource) error {
+
+	switch condition {
+	case v1alpha1.SamplesExist:
+
+	}
+
```

Comment:
Was  this you deciding you'd rather have a switch vs. an if check, but you did not complete the conversion ?

Reply:
other way around. I started out writing a switch.  Anyway it should have been removed, thanks.

Reply:
(actually it was me debating where to put the logic)

Reply:
10-4

Reply:
fixed